EvalServer/src/crud/deepeval_orgs.py

```
@@ -17,51 +19,55 @@ async def create_org(
     if row:
         return {
             "id": row["id"],
             "name": row["name"],
+            "tenant": row["tenant"],
             "createdAt": row["created_at"].isoformat() if row["created_at"] else None,
         }
     # If conflict (existing), fetch it
     res2 = await db.execute(
-        text(f'SELECT id, name, created_at FROM "{schema}".deepeval_organizations WHERE name=:name'),
-        {"name": name},
+        text(f'SELECT id, name, tenant, created_at FROM "{schema}".deepeval_organizations WHERE name=:name AND tenant=:tenant'),
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)